runelite-client/src/main/java/net/runelite/client/plugins/banktags/TagManager.java

```diff
@@ -79,6 +80,31 @@ String getTagString(int itemId, boolean variation)
 		return config;
 	}
 
+	public int getQuantityOfItemForTag(String tag, int itemId, boolean variation)
```

Comment:
This can just return the String from config since it's already converted to a string when you set it.